src/styles/partials/components/_modal.scss

```diff
@@ -0,0 +1,153 @@
+.dg_modal {
+  .dg_icon-link {
+    padding: 0;
+    margin: 0;
+    border: 0;
+  }
+
+  .dg_modal__inner {
+    overflow: auto;
+    position: relative;
+    padding: $padding-largest;
+    margin-top: $margin-small;
+    margin-bottom: $margin-small;
+    width: auto;
+    height: 500px;
+    opacity: 1;
+    color: $white;
+
+    p {
+      letter-spacing: 0;
+      color: $white;
+      opacity: 1;
+      text-align: center;
+      font: italic 28px/44px $default-body-font;
+      margin-bottom: 60px;
+    }
+  }
+}
+@keyframes fade-in {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+@keyframes appear {
+  from {
+    transform: translate(-50%, -40%);
+    opacity: 0;
+  }
+  to {
+    transform: translate(-50%, -50%);
+    opacity: 1;
+  }
+}
+[data-a11y-dialog-native] {
+  .dialog-overlay {
+    display: none;
+  }
+}
+dialog[open] {
+  display: block;
+  background: $pale-blue;
+}
+
+.dialog[aria-hidden="true"] {
+  display: none;
+}
+.dialog-overlay {
+  z-index: 900;
+  background-color: rgba(0, 0, 0, 0.66);
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  background-color: rgba(43, 46, 56, 0.9);
+}
+dialog {
+  &::backdrop {
+    background-color: rgba(0, 0, 0, 0.66);
+  }
+}
+.dialog-content {
+  background-color: rgb(255, 255, 255);
+  z-index: -1;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  -webkit-transform: translate(-50%, -50%);
+  -ms-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
+  margin: 0;
+  padding: 1em;
+  max-width: 90%;
+  width: 600px;
+  border-radius: 2px;
+  border: $border-width-small solid $blue-lightest;
+}
+.dialog {
+  &:not([aria-hidden="true"]) {
+    > .dialog-overlay {
+      animation: fade-in 200ms 1 both;
+    }
+    > .dialog-content {
+      animation: appear 400ms 150ms 1 both;
+    }
+  }
+  h1 {
+    margin: 0 0 10px 0;
+    font-size: 1.25em;
+    text-align: left;
+    color: black;
```

Comment:
I suspect we don't need many of these styles, but I'm not sure, I'll get back to you.